coveo/ui-kit
# chore: added multi-root workspaces config for vscode

https://coveord.atlassian.net/browse/KIT-2649

Part of the 20% initiative!

In this PR, I added a [multi-root workspaces](https://code.visualstudio.com/docs/editor/multi-root-workspaces) configuration to the repo and I added an extension for unit testing some packages.

To test this PR, I highly recommend that you:
1. Checkout this branch
2. When prompted, install recommended extensions
3. Open the multi-root workspaces (can be done by running `code ui-kit.code-workspace` in the terminal).
4. Press CMD+P (macOS) or CTRL+P (Windows)
5. Search for "category-field-suggestions.test.ts"
6. Make any change to the file. Tests should auto-run.
7. Look at the "Testing" pane on the left-hand-side of vscode. You should see 4 projects and their tests.

I've been using a similar config (excluding the Jest part) for maybe about 2 months and it worked well.
